apps/emqx_utils/rebar.config

```diff
@@ -6,7 +6,8 @@
 
 {deps, [
     {jiffy, {git, "https://github.com/emqx/jiffy", {tag, "1.0.6"}}},
-    {emqx_http_lib, {git, "https://github.com/emqx/emqx_http_lib.git", {tag, "0.5.3"}}}
+    {emqx_http_lib, {git, "https://github.com/emqx/emqx_http_lib.git", {tag, "0.5.3"}}},
+    {erlang_qq, {git, "https://github.com/k32/erlang_qq.git", {ref, "beab7523"}}}
```

Comment:
pin a tag ?

Reply:
There are no tags right now. I'll see if we can cut one quickly.